README.md

```diff
@@ -141,32 +152,42 @@ You can customize prometheus-service with the following environment variables:
 
 - Every service provides the following metrics for its corresponding scrape job:
     - http_response_time_milliseconds (Histogram)
     - http_requests_total (Counter)
 
-      This metric has to contain the `status` label, indicating the HTTP response code of the requests handled by the service.
-      It is highly recommended that this metric also provides a label to query metric values for specific endpoints, e.g. `handler`.
+      This metric has to contain the `status` label, indicating the HTTP response code of the requests handled by the
+      service. It is highly recommended that this metric also provides a label to query metric values for specific
+      endpoints, e.g. `handler`.
 
-      An example of an entry would look like this: `http_requests_total{method="GET",handler="VersionController.getInformation",status="200",} 4.0`
+      An example of an entry would look like
+      this: `http_requests_total{method="GET",handler="VersionController.getInformation",status="200",} 4.0`
 
 - Based on those metrics, the queries for the SLIs are built as follows:
 
-    - **throughput**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
-    - **error_rate**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary",status!~'2..'}[<test_duration_in_seconds>s]))/sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
-    - **response_time_p50**: `histogram_quantile(0.50, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
-    - **response_time_p90**: `histogram_quantile(0.90, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
-    - **response_time_p95**: `histogram_quantile(0.95, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
+    - **
+      throughput**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
+    - **
+      error_rate**: `sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary",status!~'2..'}[<test_duration_in_seconds>s]))/sum(rate(http_requests_total{job="<service>-<project>-<stage>-canary"}[<test_duration_in_seconds>s]))`
+    - **
+      response_time_p50**: `histogram_quantile(0.50, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
+    - **
+      response_time_p90**: `histogram_quantile(0.90, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
+    - **
+      response_time_p95**: `histogram_quantile(0.95, sum(rate(http_response_time_milliseconds_bucket{job='<service>-<project>-<stage>-canary'}[<test_duration_in_seconds>s])) by (le))`
```

Comment:
Pretty sure those changes were not intended, but done by the IDE :D however, they kill the formatting:
![image](https://user-images.githubusercontent.com/56065213/179496120-b7247000-5747-4a2a-b8a0-20fc6747c95f.png)

Reply:
Sorry, I forgot to render the file after the formatting. I just reversed the changes.